pyproject.toml

```
@@ -0,0 +1,31 @@
+[project]
+name = "ngen-forcings"
+authors = [
+    { name = "Jason Ducker", email = "jason.ducker@noaa.gov" }
+]
+description = "NextGen Forcings Engine"
+readme = "README.md"
```

Just to be on the safe side, I would recommend specifying a context type here. That will help software that consume wheel and source distribution archives property display the readme.
```suggestion
readme = { file = "README.md", content-type = "text/markdown" }
```

---

While you are at it, I would also add the license too (goes in `project` section).

```
license = { file = "LICENSE" }
```

---

Awesome, thank you! Added in 8d23251